workflow_scripts/protobuf/build_protobuf_unix.sh

```diff
@@ -23,17 +24,25 @@ fi
 # Build protobuf from source with -fPIC on Unix-like system
 ORIGINAL_PATH=$(pwd)
 cd ..
-wget https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protobuf-cpp-3.21.12.tar.gz
-tar -xvf protobuf-cpp-3.21.12.tar.gz
-cd protobuf-3.21.12
+wget https://github.com/abseil/abseil-cpp/releases/download/20230802.2/abseil-cpp-20230802.2.tar.gz
```

Comment:
https://protobuf.dev/reference/cpp/abseil/

I assume that would be the absl version that matches the minimal protobuf version? Does that also fit the newer protobuf versions? (So far I would have assumed that the corresponding version is “included”)

Reply:
I used the absl version that contained in the third-party of the protobuf version. I'm not sure whether newer ones are compatible.